tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -42,42 +47,75 @@
 kubeconfig_path = DATA_SUBDIR + "/kube_config_cluster-ha-filled.yml"
 export_cmd = "export KUBECONFIG=" + kubeconfig_path
 
+rke_config = {"authentication": {"type": "authnConfig", "strategy": "x509"},
```

Comment:
We have an rke_config defined in common.py. Can we not use that here instead?

Reply:
I import the `rek_config` from the `test_rke_cluster_provisioning.py` instead of moving it to `common.py` because I see the `rke_config` is imported into multiple files and I don't want to change all of them in this PR. 
I can have another PR for moving the `rke_config` and need to validate it is not breaking anything.